core/client/src/test/java/alluxio/client/file/UnderFileSystemFileInStreamTest.java

```diff
@@ -139,7 +144,5 @@ public void readToEOFTest() throws Exception {
       Assert.assertEquals((byte) i, b[i]);
     }
     Assert.assertEquals(-1, mTestStream.read());
```

Comment:
Right, the -1 is the external functionality and the `mEOF` is internal. We should only test external.